plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -453,6 +455,11 @@ class Errata(UnitMixin, ContentUnit):
 
     SERIALIZER = serializers.Errata
 
+    # Erratum fields that can be updated
+    erratum_fields_update = ('status', 'updated', 'description', 'pushcount', 'references',
```

Comment:
Consider renaming this to `mutable_erratum_fields` which would allow the comment above to be removed. I think the idea that a field is mutable means it can be updated.